app/Actions/Photo/Archive.php

```diff
@@ -19,7 +22,37 @@ class Archive
 {
 	use Constants;
 
-	private $badChars;
+	public const LIVEPHOTOVIDEO = 'LIVEPHOTOVIDEO';
```

Comment:
Considering refactoring when 8.1 is live : https://wiki.php.net/rfc/enumerations :)

Reply:
Also wondering if those should be moved to app/Actions/Photo/Extensions/Constants.php

Reply:
I haven't written that part of code. That is old code. But yes, this should be refactored someday.